cli/src/utils/cli-flags.ts

```diff
@@ -3,21 +3,23 @@ import { error } from '@oclif/errors';
 import { isISO8601, isInt } from 'validator';
 import ms from 'ms';
 
-export type MinMaxIntFlagOptions = Parameters<(typeof flags.integer)>[0] & {
+export type MinMaxIntFlagOptions = Parameters<(typeof flags.integer)>[0] & Partial<{
```

Comment:
If your'e not calling `flags.integer` I guess `Parameters<(typeof flags.integer)>[0]` should be changed also.
BUT, I don't understand why you're changing the code here at all...